### PR TITLE
 Cloth & biogenerator stuff

### PR DESCRIPTION
**Pull request by DogZeroX**
_That's pretty much everything... Just ignore the first two commits_

Edit: In the end, I did more stuff.

### CODE REVIEW

**Comment by king5327** [comment]
The song file seems to not be working right? I can't open it if I try downloading it.

**Comment by king5327** [comment]
Update: It only works in VLC. I question the format.

**Comment by king5327** [comment]
Okay it seems fine, never mind the past two comments.

**Comment by DogZeroX** [comment]
Wack

File: code/game/machinery/biogenerator.dm [hunk 1]
```diff
@@ -24,6 +24,7 @@
 			list(name="Left4Zed, 30u", cost=60, reagent="left4zed"),
 			list(name="Robust Harvest, 30u", cost=75, reagent="robustharvest"),
 		"Leather",
+		    list(name="Leather sheet", cost=50, pat=/obj/item/stack/material/cloth),
```
Comment (DogZeroX) [inline code/game/machinery/biogenerator.dm]: This looks weird, I think I did a bad

File: code/datums/craft/recipes/clothing.dm [hunk 2]
```diff
@@ -39,4 +39,52 @@
...
+	steps = list(
+		list(/obj/item/weapon/reagent_containers/glass/bucket, 1, "time" = 30),
+		list(QUALITY_CUTTING, 15, 120),
+		list(QUALITY_DRILLING, 10, 60),
+		list(CRAFT_MATERIAL, 6, MATERIAL_STEEL),
+		list(/obj/item/stack/cable_coil, 4)
+	)
+
+// Add some steel to gloves. Simple.
+/datum/craft_recipe/clothing/combat_gloves
```
Comment (DogZeroX) [inline code/datums/craft/recipes/clothing.dm]: Postdata: I didn't add the gloves and helmet. That's Nopeman's commits. I did a dumb and had to copypaste this stuff. *kisses @NopemanMcHalt*

**Comment by DogZeroX** [comment]
**REEEEEEEEEEEEEEEEEEEEE**

File: code/datums/craft/recipes/clothing.dm [hunk 3]
```diff
@@ -72,3 +72,22 @@
 		list(/obj/item/clothing, 1, time = 30),
 		list(QUALITY_CUTTING, 10, 60)
 	)
+	
+//Dunno if I'm doing this right. Also, addition above big gay. Janiboi is a closet gay
+/datum/craft_recipe/clothing/blindfold
+    name = "blindfold"
```
Comment (Trilbyspaceclone) [inline code/datums/craft/recipes/clothing.dm]: Replaces these spaces with tabs

File: code/datums/craft/recipes/clothing.dm [hunk 4]
```diff
@@ -72,3 +72,22 @@
...
+	result = /obj/item/clothing/glasses/blindfold
+	steps = list(
+		list(/obj/item/stack/material/cloth, 5, time = 15),
+		list(QUALITY_CUTTING, 10, 60)
+	)
+
+/datum/craft_recipe/clothing/muzzle
+    name = "muzzle"
```
Comment (Trilbyspaceclone) [inline code/datums/craft/recipes/clothing.dm]: replaces these spaces with tabs

**Comment by DogZeroX** [comment]
@Trilbyspaceclone https://www.youtube.com/watch?v=jnOSkB7O2ug

**Comment by APWill** [comment]
PR Review Team recommends APPROVAL. Seems like a nice little PR that adds more crafting capabilities, a song that's actually pretty fitting for the menu, and good quotes. Thanks for the PR!

File: strings/quotes.txt [hunk 5]
```diff
@@ -99,4 +99,5 @@
 "Life, and intelligence, do not mix very well." - Bruce Sterling, Schismatrix
 "The Zaibatsu recognizes one civil right: the right to death." - Bruce Sterling, Schismatrix
 "Posthumanism offers fluidity and freedom, and a metaphysic daring enough to think a whole world into life." - Bruce Sterling, Schismatrix
-"In terms of morals there is no such thing as �state.� Just men. Individuals. Each responsible for his own acts." - Robert A. Heinlein, The Moon is a Harsh Mistress
\ No newline at end of file
+"In terms of morals there is no such thing as �state.� Just men. Individuals. Each responsible for his own acts." - Robert A. Heinlein, The Moon is a Harsh Mistress
+"It's only game, why you heff to be mad?" - Ilya Bryzgalov, Ex-Hockey player
```
Comment (king5327) [inline strings/quotes.txt]: No.

Comment (DogZeroX) [reply strings/quotes.txt]: *grumbles!*

**Comment by DogZeroX** [comment]
@APWill Thanks. Now give me the **S U C C**

**Comment by DogZeroX** [comment]
@king5327 Remove the change request, please. PD: Sorry if you're getting your emails spammed.

**Comment by DogZeroX** [comment]
Update: Blindfold doesn't exist in the code.

**Comment by Kazkin** [comment]
Have we gotten this one fully working? Seems alright in terms of what its trying to do.

**Comment by DogZeroX** [comment]
It works for me, tried on local.

**Comment by DogZeroX** [comment]
@Kazkin

**Comment by Kazkin** [comment]
Hm, if its all functioning and the requested changes are implemented then I say we should approve it but verify if its working after. We can just revert if its broke or bug fix it.

**Comment by DogZeroX** [comment]
Gotta change some cratings after this PR is merged, like clothes and that. Maybe makeshift bandages

**Comment by Kazkin** [comment]
Makeshift bandages with cloth is already a thing, currently.

**Comment by DogZeroX** [comment]
Really? I don't remember.

**Comment by Kazkin** [comment]
Aye, under misc crafting in the crafting menu you can use 2 cloth to produce a bandage.

**Comment by DogZeroX** [comment]
Mhm. Gotta make a crafting to obtain cloth by cutting clothes too

**Comment by Kazkin** [comment]
That would be a fantastic PR if you could use a knife to turn any jumpsuit or over wear into cloth.

**Pull request closed without merging**